monitor/sth_fetcher.go

```diff
@@ -290,7 +285,8 @@ func (f *sthFetcher) verifySTHConsistency(firstSTH, secondSTH *ct.SignedTreeHead
 
 	// Verify the consistency proof. If the proof fails to verify then publish an
 	// increment to the `sthInconsistencies` stat
-	if err := f.verifier.VerifyInclusion(
+	if err := f.verify(
```

Comment:
I actually had the exact same question while I was writing this PR. I was very confused but decided to stick with it and then completely forgot to ask when I uploaded it. Since I haven't done much of anything in this repo previously -- @jsha do you know what's going on here, and why this uses the `VerifyFoo` function that seems to be the opposite of what it claims to be doing?

Reply:
Here are the issue and PR for this:

https://github.com/letsencrypt/ct-woodpecker/issues/4
https://github.com/letsencrypt/ct-woodpecker/pull/37

I believe yes, this is supposed to verify consistency between two STHs, not inclusion of a specific leaf. I haven't dug in to see whether I would currently expect it to be broken, but we should check out stats and see if it is.

Reply:
Looks like it accidentally got changed from `VerifyConsistency` to `VerifyInclusion` in the update+refactoring about a month ago: https://github.com/letsencrypt/ct-woodpecker/pull/119/files#diff-4757948af9ae13198a85a88d03dad91762b70adeb7106e22f8b13e39e6919ce4R293

Uploading a patchset which changes it back.

Reply:
Ooh nice catch, thanks.